pkgs/applications/misc/paperjam/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib , stdenv, fetchgit, qpdf, libpaper, asciidoc, }:
```

Comment:
```suggestion
{ lib 
, stdenv
, fetchgit
, asciidoc
, libpaper
, qpdf
}:
```

Reply:
I consider this to be a matter of personal taste, not a nixpkgs rule and your version is wasting precious on-screen lines of text, but whatever, I wan't this merged after ½ a year